docs/admin/addons.rst

```diff
@@ -236,26 +232,32 @@ Squash Git commits
 
 Squash Git commits prior to pushing changes. You can choose one of following modes:
 
+.. versionadded:: 3.4
+
 * All commits into one
 * Per language
 * Per file
+
+.. versionadded:: 3.5
+
 * Per author
 
-Original commit messages are kept, but authorship is lost unless "Per author" is selected or the commit message is customized to include it.
+Original commit messages are kept, but authorship is lost unless "Per author" is selected, or
+the commit message is customized to include it.
 
 .. _addon-weblate.json.customize:
 
 Customize JSON output
 ---------------------
 
-Allows to customize JSON output behavior, for example indentation or sorting.
+Allows adjusting JSON output behavior, for example indentation or sorting.
 
 .. _addon-weblate.properties.sort:
 
 Formats the Java properties file
 --------------------------------
 
-This addon sorts the Java properties file.
+This addon sorts the Java properties file, if present.
```

Comment:
Why if present? The addon is installable only if the component is using Java properties.